dev/com.ibm.ws.concurrent.persistent_fat_demo_timers/fat/src/com/ibm/ws/concurrent/persistent/fat/demo/timers/DemoTimerTest.java

```diff
@@ -56,6 +56,15 @@ public static void setUp() throws Exception {
         //Get driver name
         server.addEnvVar("DB_DRIVER", DatabaseContainerType.valueOf(testContainer).getDriverName());
 
+        if (DatabaseContainerType.valueOf(testContainer) == DatabaseContainerType.Oracle) {
+            //Type=javax.sql.ConnectionPoolDataSource to avoid
+            //  ORA-02089: COMMIT is not allowed in a subordinate session
+            //  when application tries to create a new table using oracle.
+            server.addEnvVar("DS_TYPE", "javax.sql.ConnectionPoolDataSource");
```

Comment:
The fact that it is possible to run the test with a one-phase only data source tells me that we need to add/update a scenario to cover multiple transactional resources.  It's fine to deliver this for now, be we need a follow-on work item to cover that.  It should be an easy update, just tweak the `@Resource` for the DataSource in AutomaticDatabase timer.  But then the create table code will need to be moved so that XADataSource can be used.

Reply:
Yes, I will come back to this in another PR.